training/3-node-local-insecure-cluster-load-gen.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Catch Up&#58; 3-Node, Local, Insecure Cluster w/ Load Generator
+summary: Get a 3-node, local, insecure CockroachDB cluster up and running quickly with a load generator.
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+If you weren't in the training for the session where we set this, this guide shows you how to set up a CockroachDB cluster:
+
+- With 3 nodes
+- Locally
+- *Without* SSL encryption
+- With a load generator
+
+## Step 1. Install prerequisites
+
+In this tutorial, you'll use CockroachDB, the HAProxy load balancer, and CockroachDB's version of the YCSB load generator, which requires Go. Before you begin, make sure these applications are installed:
+
+- Install the latest version of [CockroachDB](/stable/install-cockroachdb.html).
+- Install [HAProxy](http://www.haproxy.org/). If you're on a Mac and using Homebrew, use `brew install haproxy`.
+- Install [Go](https://golang.org/dl/). If you're on a Mac and using Homebrew, use `brew install go`.
+- Install the [CockroachDB version of YCSB](https://github.com/cockroachdb/loadgen/tree/master/ycsb): `go get github.com/cockroachdb/loadgen/ycsb`
+
+Also, to keep track of the data files and logs for your cluster, you may want to create a new directory (e.g., `mkdir crdb-training`) and start all your nodes in that directory.
+
+## Step 2. Start a local 3-node cluster
+
+If you've already [started a local cluster](/stable/start-a-local-cluster.html), the commands for starting nodes should be familiar to you. The new flag to note is [`--locality`](/stable/configure-replication-zones.html#descriptive-attributes-assigned-to-nodes), which accepts key-value pairs that describe the topography of a node. In this case, you're using the flag to specify that the first 3 nodes are running on cloud 1.
```

Comment:
The --locality flag is not used below, as far as I can tell.

Reply:
Done.